sphinx/checks.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
```

Comment:
Why are you moving this to under sphinx package? This file will be installed to all users' environment. I feel very strange. Is that really needed?

Reply:
I just checked and it seems this file need to be in a module. I can either make `utils` a module, but that seems silly. Alternatively we can just distribute this. I don't imagine it would be any harm, to be honest.

Reply:
Sure, I understand this does not harm anybody. But, personally, I don't like to export meaningless packages to users. I believe this is only used for maintenance.That means it does not bring worth to users. So I'd like not to include it to the release package.
Could you move this outside of package?

Reply:
I can exclude it from the package via `MANIFEST.in`. Before I do this whoever, I should point out that [we package both the `tests` and `utils` folders](https://github.com/sphinx-doc/sphinx/blob/1.6/MANIFEST.in#L26-L27). If we are packaging these, is there any reason _not_ to package this file?

Reply:
Sorry, my words are wrong. It should be packaged as tarball, but not be installed as python package (`sphinx.checks`).

Reply:
Done now